columnflow/columnar_util.py

```diff
@@ -2654,6 +2723,16 @@ def read_coffea_parquet(
             "read_dictionary" not in read_options["parquet_options"]
         ):
             read_dictionary = [Route(s).string_column for s in read_columns]
+
+            # add names prefixed with an 'n' to the list of columns to read
+            # (needed to construct the nested list structure of jagged columns)
+            maybe_jagged_fields = {Route(s)[0] for s in read_columns}
+            read_dictionary.extend(
+                f"n{field}"
+                for field in maybe_jagged_fields
+                if field != "GenPart"
```

Comment:
Is `GenPart` still a special case with the newest software deps?

Reply:
Good point. I think the issue is solved now, I just forgot to remove the `if` condition here. I'll look into this.

Reply:
Done